common/tier4_planning_rviz_plugin/include/tier4_planning_rviz_plugin/path/display_base.hpp

```diff
@@ -232,6 +201,43 @@ class AutowarePathBaseDisplay : public rviz_common::MessageFilterDisplay<T>
   }
 
 protected:
+  std::unique_ptr<Ogre::ColourValue> gradation(
+    const QColor & color_min, const QColor & color_max, const double ratio)
+  {
+    std::unique_ptr<Ogre::ColourValue> color_ptr(new Ogre::ColourValue);
+    color_ptr->g =
+      static_cast<float>(color_max.greenF() * ratio + color_min.greenF() * (1.0 - ratio));
+    color_ptr->r = static_cast<float>(color_max.redF() * ratio + color_min.redF() * (1.0 - ratio));
+    color_ptr->b =
+      static_cast<float>(color_max.blueF() * ratio + color_min.blueF() * (1.0 - ratio));
+
+    return color_ptr;
+  }
```

Comment:
I think you can move this function back up since they don't use any of the class member variables.

Reply:
Applied with: https://github.com/autowarefoundation/autoware.universe/pull/8972/commits/d8ba16f9a28d27cc156a64f8cd64e15b0e96d58c